olive/passes/onnx/qnn_preprocess.py

```diff
@@ -47,7 +99,18 @@ def _run_for_config(
         from onnxruntime.quantization.execution_providers.qnn import qnn_preprocess_model
 
         output_model_path = resolve_onnx_path(output_model_path, Path(model.model_path).name)
-        modified = qnn_preprocess_model(model.model_path, output_model_path, **config)
+
+        modified = qnn_preprocess_model(
+            model.model_path,
+            output_model_path,
+            fuse_layernorm=config["fuse_layernorm"],
+            save_as_external_data=config["save_as_external_data"],
+            all_tensors_to_one_file=config["all_tensors_to_one_file"],
+            external_data_size_threshold=config["size_threshold"],
+            external_data_convert_attribute=config["convert_attribute"],
+            inputs_to_make_channel_last=config["inputs_to_make_channel_last"],
```

Comment:
Won't this cause key error for ort < 1.18? If we only want to support the nightly version and there is no use case for version 1.17, I think we should just make 1.18 the required version. 

Also, I don't think we should have `default_config` be conditional for ort version. We don't have any passes that do that + cache is agnostic to ort version currently. 
We could add the new options directly and ignore them with some logs for lower versions (if we want to keep support for lower version)

Reply:
@jambayk, you are right. From the time being, the default_config doesn't support conditional.